src/bin/NodeOptions.ts

```diff
@@ -72,14 +73,16 @@ export default class NodeOptions {
   /**
   * Parses the commander options and returns an Options object.
   * @param options Options as they are given by commander.
+  * @param chainId Chain id.
   * @returns The parsed options with defaults for options that are missing from the command line.
   */
-  public static parseOptions(options): NodeOptions {
+  public static parseOptions(options, chainId): NodeOptions {
+    const chainIdNumber = NodeFactory.getChainId(chainId);
```

Comment:
Seeing `NodeFactory` here, I am just slightly concerned that responsibilities are somewhat not clearly separated. Could probably be changed later, though. What do you think @deepesh-kn ?

Reply:
Created ChainInfo.js, that includes the following functions now:
- `officialIdentifiers()`
- `chainInfo()`
- `getChainId()`